Create a new monitoring enterprise doc (#3078)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/monitor-enterprise/monitor-with-cloud.md

  Monitor your InfluxDB Enterprise instance using InfluxDB Cloud and
  a pre-built InfluxDB template.
menu:
  enterprise_influxdb_1_9:
    parent: Monitor InfluxDB
    name: Monitor with Cloud
weight: 102
---

Use [InfluxDB Cloud](/influxdb/cloud/), the [InfluxDB Enterprise 1.x Template](https://github.com/influxdata/community-templates/tree/master/influxdb-enterprise-1x), and Telegraf to monitor one or more InfluxDB Enterprise instances.

Do the following:

1. [Review requirements](#review-requirements)
2. [Install the InfluxDB Enterprise Monitoring template](#install-the-influxdb-enterprise-monitoring-template)
3. [Set up InfluxDB Enterprise for monitoring](#set-up-influxdb-enterprise-for-monitoring)
4. [Set up Telegraf](#set-up-telegraf)
5. [View the Monitoring dashboard](#view-the-monitoring-dashboard)
6. (Optional) [Alert when metrics stop reporting](#alert-when-metrics-stop-reporting)
7. (Optional) [Create a notification endpoint and rule](#create-a-notification-endpoint-and-rule)
8. (Optional) [Monitor with InfluxDB Insights and Aware](#monitor-with-influxdb-insights-and-aware)

## Review requirements

Before you begin, make sure you have access to the following:

>  - InfluxDB Cloud account ([sign up for free here](https://cloud2.influxdata.com/signup))
>  - Command line access to a machine [running InfluxDB Enterprise 1.x](/enterprise_influxdb/v1.9/introduction/install-and-deploy/) and permissions to install Telegraf on this machine
>  - Internet connectivity from the machine running InfluxDB Enterprise 1.x and Telegraf to InfluxDB Cloud
>  - Sufficient resource availability to install the template. InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits)

Reviewer: ```suggestion
 - An InfluxDB Cloud account. ([Sign up for free here](https://cloud2.influxdata.com/signup)).
 - Command line access to a machine [running InfluxDB Enterprise 1.x](/enterprise_influxdb/v1.9/introduction/install-and-deploy/) and permissions to install Telegraf on this machine.
 - Internet connectivity from the machine running InfluxDB Enterprise 1.x and Telegraf to InfluxDB Cloud.
 - Sufficient resource availability to install the template. (InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits).)
```

## Install the InfluxDB Enterprise Monitoring template

> The InfluxDB Enterprise Monitoring template includes a Telegraf configuration that sends InfluxDB Enterprise metrics to an InfluxDB endpoint and a dashboard that visualizes the metrics.

Reviewer: ```suggestion
The InfluxDB Enterprise Monitoring template includes a Telegraf configuration that sends InfluxDB Enterprise metrics to an InfluxDB endpoint, and a dashboard that visualizes the metrics.
```

   2. Copy the `urls`, `token`, `organization`, and `bucket` and close the window. 
   3. Click **monitoring-enterprise-1.x**. 
   4. Replace `urls`, `token`, `organization`, and `bucket` under `outputs.influxdb_v2` with your InfluxDB Cloud account information. Alternatively, store this information in your environment variables and include the environment variables in your configuration.

      {{% note %}}
To ensure the InfluxDB Enterprise monitoring dashboard can display the recorded metrics, set the destination bucket name to `enterprise_metrics` in your `telegraf.conf`.
      {{% /note %}}

   5. Add the [Prometheus input plugin](https://github.com/influxdata/telegraf/blob/release-1.19/plugins/inputs/prometheus/README.md) to your `telegraf.conf`. Specify your your InfluxDB Enterprise URL(s) in the `urls` parameter. For example:
   
    {{< keep-url >}}
     ```toml
     [[inputs.prometheus]]
       urls = ["http://localhost:8086/metrics"]
       username = "$INFLUX_USER"
       password = "$INFLUX_PASSWORD"
     ``` 
     

>      If you're using unique URLs or have security set up for your `/metrics` endpoint, configure those options here and save the updated configuration.

Reviewer: ```suggestion
     If you're using unique URLs or have authentication set up for your `/metrics` endpoint, configure those options here and save the updated configuration.
```

      For more information about customizing Telegraf, see [Configure Telegraf](/{{< latest "telegraf" >}}/administration/configuration/#global-tags).
4. Click **Save Changes**.

## Set up Telegraf

Set up Telegraf to scrape metrics from InfluxDB Enterprise to send to your InfluxDB Cloud account.

On each InfluxDB Enterprise instance you want to monitor, do the following:

1. Go to **Load Data > Telegraf** [in your InfluxDB Cloud account](https://cloud2.influxdata.com/).
2. Click **Setup Instructions** under **monitoring-enterprise-1.x**. 
3. Complete the Telegraf Setup instructions. If you are using environment variables, set them up now. 

      {{% note %}}

> For your API token, generate a new token or use an existing All Access token. If you run Telegraf as a service, edit your init script to set the environment variable and ensure its available to the service.

Reviewer: ```suggestion
For your API token, generate a new token or use an existing All Access token. If you run Telegraf as a service, edit your init script to set the environment variable and ensure that it's available to the service.
```

2. [Create a notification endpoint and rule](#create-a-notification-endpoint-and-rule) to receive notifications when your deadman check is triggered.

### Customize the deadman check

1.  To view the deadman check, click **Alerts** in the navigation bar of your **InfluxDB Cloud** account.

    {{< nav-icon "alerts" >}}

2. Choose a InfluxDB OSS field or create a new OSS field for your deadman alert:
   1. Click **{{< icon "plus" >}} Create** and select **Deadman Check** in the dropown menu.
   2. Define your query with at least one field.
   3. Click **Submit** and **Configure Check**.
   When metrics stop reporting, you'll receive an alert.
3. Start under **Schedule Every**, set the amount of time to check for data.
4. Set the amount of time to wait before switching to a critical alert.
5. Save the Check and click on **View History** of the Check under the gear icon to verify it is running.  

## Create a notification endpoint and rule

To receive a notification message when your deadman check is triggered, create a [notification endpoint](#create-a-notification-endpoint) and [rule](#create-a-notification-rule). 

### Create a notification endpoint 

> InfluxData supports different endpoints: Slack, PagerDuty, and HTTP. Slack is free for all users, while PagerDuty and HTTP are exclusive to the Usage-Based Plan. 

Reviewer: ```suggestion
InfluxDB Cloud supports different endpoints: Slack, PagerDuty, and HTTP. Slack is free for all users, while PagerDuty and HTTP are exclusive to the Usage-Based Plan. 
```
